package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "stick",
-    "version": "0.4.0",
+    "version": "0.5.0",
```

Comment:
yes. we try to follow semver and it recommends increasing minor version for 0.x.y releases.